cli/app.go

```diff
@@ -2977,6 +2993,39 @@ This won't work unless you have an existing installation of our GitHub app on yo
 							Usage: "remote user's home directory. only necessary if you're targeting a remote machine where $HOME is not /root",
 							Value: "~",
 						},
+						&cli.BoolFlag{
+							Name:  moduleBuildFlagCloudBuild,
+							Usage: "Run the build script of the module using cloud build instead of locally. The file will be downloaded to the build.path field in meta.json",
+						},
+						&cli.StringFlag{
+							Name:        moduleBuildFlagId,
+							Usage:       "Provide with --cloud-build to reload with a previous build instead of generating a new one",
```

Comment:
this is smart to have

do all reload builds also get a unique ID?

Reply:
Yep, all reload builds get a unique build ID. Though I think this might actually only be working with proper semvers, need to look into it!

Reply:
okay to postpone / not do